Plugins/GRpluginStands.txt

```diff
@@ -1739,178 +2020,339 @@ PRIORITY:+1
+COORD:N063.27.19.149:E010.54.55.006
+COORD:N063.27.16.837:E010.54.54.966
 PRIORITY:-1
 CODE:D
 ADEP:EN
 
 //STAND 32
-STAND:ENVA:32:N063.27.17.580:E010.54.56.820:18
+STAND:ENVA:32
+COORD:N063.27.16.860:E010.54.55.333
+COORD:N063.27.16.812:E010.54.58.114
+COORD:N063.27.19.163:E010.54.58.177
+COORD:N063.27.19.146:E010.54.55.362
+COORD:N063.27.16.860:E010.54.55.333
 CODE:C
 ADEP:EN
 
 //STAND 33
-STAND:ENVA:33:N063.27.17.580:E010.54.59.750:18
+STAND:ENVA:33
+COORD:N063.27.16.835:E010.54.58.480
+COORD:N063.27.16.793:E010.55.01.019
+COORD:N063.27.19.150:E010.55.01.066
+COORD:N063.27.19.153:E010.54.58.477
+COORD:N063.27.16.835:E010.54.58.480
 CODE:C
 ADEP:EN
 
 //STAND 34
-STAND:ENVA:34:N063.27.17.580:E010.55.02.680:18
+STAND:ENVA:34
+COORD:N063.27.16.823:E010.55.01.370
+COORD:N063.27.16.801:E010.55.03.977
+COORD:N063.27.19.158:E010.55.04.010
+COORD:N063.27.19.154:E010.55.01.422
+COORD:N063.27.16.823:E010.55.01.370
 CODE:C
 ADEP:EN
 
 //STAND 35
-STAND:ENVA:35:N063.27.17.580:E010.55.05.600:18
+STAND:ENVA:35
+COORD:N063.27.16.823:E010.55.04.286
+COORD:N063.27.16.795:E010.55.06.865
+COORD:N063.27.19.135:E010.55.06.987
+COORD:N063.27.19.141:E010.55.04.311
+COORD:N063.27.16.823:E010.55.04.286
 CODE:C
 ADEP:EN
 
 //STAND 36
-STAND:ENVA:36:N063.27.17.570:E010.55.08.500:18
+STAND:ENVA:36
+COORD:N063.27.16.817:E010.55.07.204
+COORD:N063.27.16.776:E010.55.09.984
+COORD:N063.27.19.133:E010.55.10.046
+COORD:N063.27.19.131:E010.55.07.315
+COORD:N063.27.16.817:E010.55.07.204
 CODE:C
 ADEP:EN
 
 //STAND 37
-STAND:ENVA:37:N063.27.17.570:E010.55.11.800:18
+STAND:ENVA:37
+COORD:N063.27.16.798:E010.55.10.308
+COORD:N063.27.16.766:E010.55.13.428
+COORD:N063.27.19.119:E010.55.13.562
+COORD:N063.27.19.124:E010.55.10.403
+COORD:N063.27.16.798:E010.55.10.308
 CODE:C
 ADEP:EN
 
 //STAND 38
-STAND:ENVA:38:N063.27.18.940:E010.55.14.730:18
+STAND:ENVA:38
+COORD:N063.27.19.122:E010.55.13.931
+COORD:N063.27.19.112:E010.55.16.141
+COORD:N063.27.18.371:E010.55.16.142
+COORD:N063.27.18.388:E010.55.13.947
+COORD:N063.27.19.122:E010.55.13.931
 NOTATYP:A3*,B7*
 CODE:C
 PRIORITY:-3
 
 //STAND 39
-STAND:ENVA:39:N063.27.19.050:E010.55.18.670:18
+STAND:ENVA:39
+COORD:N063.27.18.371:E010.55.18.962
+COORD:N063.27.18.382:E010.55.16.529
+COORD:N063.27.19.130:E010.55.16.541
+COORD:N063.27.19.118:E010.55.20.094
+COORD:N063.27.18.371:E010.55.18.962
 NOTATYP:A3*,B7*
 CODE:C
 BLOCKS:40L
 PRIORITY:-2
 
 //STAND 40
-STAND:ENVA:40:N063.27.18.380:E010.55.20.900:18
+STAND:ENVA:40
+COORD:N063.27.17.993:E010.55.24.432
+COORD:N063.27.18.024:E010.55.20.146
+COORD:N063.27.18.514:E010.55.20.225
+COORD:N063.27.18.905:E010.55.21.527
+COORD:N063.27.18.868:E010.55.24.427
+COORD:N063.27.17.993:E010.55.24.432
 BLOCKS:40L
 CODE:C
 NOTADEP:EN
 
 //STAND 40L
-STAND:ENVA:40L:N063.27.18.180:E010.55.20.530:18
+STAND:ENVA:40L
+COORD:N063.27.17.793:E010.55.24.431
+COORD:N063.27.17.830:E010.55.19.651
+COORD:N063.27.18.860:E010.55.19.713
+COORD:N063.27.18.835:E010.55.24.446
+COORD:N063.27.17.793:E010.55.24.431
 BLOCKS:39,41
```

Comment:
As these are on top of each other, I would change it so it only blocks if code D is parked here - otherwise it might detect a B738 on stand 40, being parked on 40L.

`BLOCKS:39,41:D` should solve this

Reply:
Thanks! Fixed in [8ec132d](https://github.com/Vatsim-Scandinavia/ENOR-Sectorfile/pull/406/commits/8ec132d57fb0121fb902f328017e437d70bc8b65).